### ISSUE TITLE
Could not find com.apollographql.apollo:apollo-runtime:2.5.7.

### ISSUE THREAD

**Issue by lukas1** [apollographql/apollo-android#3116]
**Summary**
Updating to `2.5.7` doesn't work, Gradle can't resolve `com.apollographql.apollo:apollo-runtime:2.5.7` .

**Version**
`2.5.7`

**Comment by martinbonnin** [comment]
Indeed, it hasn't been deployed :-/. Looks like there was an incident on OSSRH around the same time: https://status.maven.org/incidents/5wx4907mjlqc. I'll rerun the deploy step manually.

**Comment by martinbonnin** [comment]
I don't want to overwhelm the Sonatype systems if they have a huge backlog of sync operations, I'll wait for their instructions before trying to release again. As a workaround, you can use the following staging repository:

```
repositories {
  maven("https://oss.sonatype.org/content/repositories/comapollographql-1169")
}
```
